### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by esethna**

### CODE REVIEW

**Comment by esethna** [comment]
Added note about needing v5.16 for away status detection. @deanwhillier please review and merge when possible

**Comment by esethna** [comment]
Added note about Catalina

**Comment by deanwhillier** [comment]
@esethna, technically the 5.14 and 5.15 dot releases also support away status updates. Since the notification stuff uses the desktop<->webapp bridge and the bridge needed to be updated for `context-isolation`, the away status support was necessarily also updated 'for free'. Apologies, I forgot to mention that to you previously.

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,7 +21,7 @@ Desktop App v4.3.0 contains multiple medium level security fixes. [Upgrading](ht
 ### Improvements
 
 #### All Platforms
- - Added support for maintaining a user's online status while the desktop app is in the background but the user is interacting with their computer.
+ - Added support for maintaining a user's online status while the desktop app is in the background but the user is interacting with their computer. Requires Mattermost Server v5.16.0 or later.
```
Comment (deanwhillier) [inline CHANGELOG.md]: Could add in here that the latest dot releases of 5.14 and 5.15 support the online status feature.

Comment (amyblais) [inline CHANGELOG.md]:
```suggestion
 - Added support for maintaining a user's online status while the desktop app is in the background but the user is interacting with their computer. Requires Mattermost Server v5.16.0, v5.15.1, v5.14.4, v5.9.5 or later.
```

Comment (esethna) [reply CHANGELOG.md]: Which versions exactly? We can add them here

Comment (esethna) [reply CHANGELOG.md]: @deanwhillier is the above accurate? I don't think the v5.9 release supports it?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,13 +2,15 @@
...
 
-- **v4.3.1, release day TBD**
+- **v4.3.2, release day TBD**
   - `CMD+Z` doesn't undo on the Mac desktop app. [MM-19198](https://mattermost.atlassian.net/browse/MM-19198)
   - Users are unable to zoom in/out except on the first server tab. [MM-19032](https://mattermost.atlassian.net/browse/MM-19032)
   - Right-click + "Copy" does not work in some instances. [MM-19324](https://mattermost.atlassian.net/browse/MM-19324)
   - Unable to exit maximized Youtube videos using the close button if the app is running in full screen. [MM-19226](https://mattermost.atlassian.net/browse/MM-19226)
   - Desktop notifications appear in sequence rather than stacking on Windows. [MM-19363](https://mattermost.atlassian.net/browse/MM-19363)
   - Clicking on notifications when using the Windows MSI (Beta) doesn't focus the app or the channel that triggered the notification. [MM-19510](https://mattermost.atlassian.net/browse/MM-19510), [MM-19511](https://mattermost.atlassian.net/browse/MM-19511)
+- **v4.3.1, released 2019-10-22**
```
Comment (amyblais) [inline CHANGELOG.md]: @esethna I added a note that v4.3.1 is released.
Same here: https://github.com/mattermost/docs/pull/3095.

File: CHANGELOG.md (hunk 1 above)
Comment (deanwhillier) [reply CHANGELOG.md]: That's correct, v5.9 doesn't support it as the feature was initially added to 5.14 so I didn't back port all the way to 5.9. The rest is correct.

Comment (deanwhillier) [reply CHANGELOG.md]:
```suggestion
 - Added support for maintaining a user's online status while the desktop app is in the background but the user is interacting with their computer. Requires Mattermost Server v5.16.0, v5.15.1, v5.14.4 or later.
```

**Pull request closed without merging**